tests/invalid_models_tests/test_models.py

```diff
@@ -909,3 +909,21 @@ def dummy_function(*args, **kwargs):
                 id='signals.E001',
             ),
         ])
+
+
+@isolate_apps('invalid_models_tests')
+class IndexTests(SimpleTestCase):
+
+    def test_pointing_to_missing_field(self):
```

Comment:
this doesn't need a separate test class, insert the test after `test_name_contains_double_underscores` which tests E024